react/controls/FullscreenButton.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { FunctionComponent } from 'react'
+import { Icon } from 'vtex.store-icons'
+
+import styles from '../styles/styles.css'
+
+export interface FullscreenButtonProps {
+  toggleFullscreenMode: () => void
+  cssHandles: any
+}
+
+const FullscreenButton: FunctionComponent<FullscreenButtonProps> = ({
+  toggleFullscreenMode,
+  cssHandles,
+}) => {
+  return (
+    <button
+      className={`${cssHandles.fullscreenButton} ${styles.button} ml4 absolute bottom-2 right-0`}
+      onClick={() => toggleFullscreenMode()}
+    >
+      <Icon id="hpa-fullscreen" />
```

Comment:
Instead of using the `Icon` component like this, you can import directly the icon you want to use. I'm guessing the one to use here is the `IconExpand` component

Reply:
Why not use a slot with default `IconExpand`? Do you agree @victorhmp ?

Reply:
Yep! That would be better